queue_and_stack/dll_stack.py

```diff
@@ -6,13 +6,20 @@ class Stack:
     def __init__(self):
         self.size = 0
         # Why is our DLL a good choice to store our elements?
-        # self.storage = ?
+        self.storage = DoublyLinkedList()
 
-    def push(self, value):
-        pass
+    def push(self, value): # add to head
```

Comment:
To better drive home the concept of a Stack it's best to add to `tail` that adds an item to the last node of the `linked list` and remove from `tail` that is the first node. You then do the reverse for a Queue.

Reply:
I used the English definition for stack and queue when implementing. For instance a queue IRL you add to the tail and remove from head, same case when dealing with a stack of papers; you add to the top of  the stack and also start removing from the top of the stack. I'll consider your suggestion when refactoring to cater for both approaches 💯.